controllers/drinks/favorite/addFav.js

```diff
@@ -0,0 +1,29 @@
+const { Recipe } = require("../../../models/recipe");
+const User = require("../../../models/user");
+const { requestError } = require("../../../services");
+
+const addFav = async (req, res) => {
+  const { _id } = req.user;
+  const cocktailId = req.body.cocktailId;
+  const cocktail = await Recipe.findById(cocktailId);
+
+  if (!cocktail) {
+    throw requestError(404, "Wrong ID");
```

Comment:
трохи інформативніше
```suggestion 
    throw requestError(404, "Cocktail is not exist");
```

Reply:
або 
```suggestion
    throw requestError(404, "Not found");
```